resource_customizations/policy.open-cluster-management.io/Policy/health.lua

```diff
@@ -1,4 +1,9 @@
 hs = {}
+if obj.spec.disabled then
+  hs.status = "Healthy"
+  hs.message = "Policy is disabled"
+  return hs
+end
```

Comment:
Would `Suspended` make more sense? From the docs:

> Suspended - the resource is suspended and waiting for some external event to resume (e.g. suspended CronJob or paused Deployment)

Reply:
@crenshaw-dev I was thinking the same thing, but we got feedback over the course of this PR that "Healthy" would be preferable. And (my own view here) particularly without an accompanying custom action in ArgoCD to re-enable it.